ui/app/mixins/window-resizable.js

```diff
@@ -0,0 +1,16 @@
+import Ember from 'ember';
+
+const { run, $ } = Ember;
+
+export default Ember.Mixin.create({
```

Comment:
I know we don't have a lot of components using this now (and maybe we won't ever?) - but would this be better as (or paired with) a service that had an array of handlers that would lazily attach the global resize handler? That way there's just one resize event handler that fires all of attached callbacks (shout out to $.Callbacks). Also this is 100% something we could do later, just wanted to put a bug in your ear about it.

Reply:
> That way there's just one resize event handler that fires all of attached callbacks

Isn't that what `jQuery.on` is already doing?

Reply:
For posterity: jQuery _is_ already doing this! - @DingoEatingFuzz went spelunking and shared this:   https://github.com/jquery/jquery/blob/master/src/event.js#L184-L216